### DISCUSSION TITLE
How to add current playing track to playlist?

### DISCUSSION THREAD

**Discussion by mmol67** [navidrome/navidrome#2232] [Q&A] [answered]
Is there a quick way to do it? Say a button in the playing banner at the botton of the page, or an action in the 3-dot menu in this banner... I can't find it.

**Comment by sphars** [comment]
You can click on the playing song title, which takes you to the album page, and from there you can add to a playlist. Not as quick but it's something.

**Reply by mmol67** [reply]
Thank you. It is what I was doing. I'm just thinking that should be a one-click way to do it, or at least a in-place action (in the 3 dot menu).

Regards

**Comment by deluan** [comment]
Unfortunately there's no easier way to add the playing track to a playlist. But I'm confused: when you say "in the 3 dot menu", what exactly do you mean? There is a Add to Playlist option in the context menu:
<img width="197" alt="Screenshot 2023-03-09 at 8 12 46 AM" src="https://user-images.githubusercontent.com/331353/224034367-7ff47f00-61f1-4bdf-9cb6-70ef50a8d798.png">

Or maybe you are talking about a different menu?

**Reply by mmol67** [reply]
Ah ok, sorry. I was visualizing a 3-dot menu in the playing song banner, that is not there.

**Comment by mmol67** [comment]
Hello again. I've been testing Substramer web (subsonic web  client) and it has a 3 dot menu in the playing track so you can do some actions with that track, including "add to playlist". I think it could be a good addition to Navidrome's web interface. And you can "discharge" the space in the playing track banner including some of the actions that now are buttons in that menu. What do you think?

**Comment by deluan** [comment, marked as answer]
This will be possible when we have a new player, maybe part of the UI rewrite I'm working on.

**Comment by kurazli** [comment]
I was just looking for the same feature while trying to save my play queue to a playlist and found this issue which seems related.
Is there any progress on this?
